pkgs/by-name/pg/pgsrip/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "pgsrip";
+  version = "0.1.11";
+  pyproject = true;
+
+  disabled = python3Packages.pythonOlder "3.9";
```

Comment:
This line is a bit useless because python3Packages will never be 3.9 or below.

Reply:
Maybe not by default, but I don't think that it hurts to include this information in any case.